src/storm/builder/DdPrismModelBuilder.cpp

```diff
@@ -34,7 +34,8 @@ namespace builder {
 template<storm::dd::DdType Type, typename ValueType>
 class ParameterCreator {
    public:
-    void create(storm::prism::Program const& program, storm::adapters::AddExpressionAdapter<Type, ValueType>& rowExpressionAdapter) {
+    void create([[maybe_unused]] storm::prism::Program const& program,
+                [[maybe_unused]] storm::adapters::AddExpressionAdapter<Type, ValueType>& rowExpressionAdapter) {
         // Intentionally left empty: no support for parameters for this data type.
```

Comment:
Shouldnt there be an error here then?

Reply:
Yeah, I was confused about this too. This is also listed in my "should be decided on by someone who knows what is going on here" markdown document for this reason, as well as the lack of NotSupportedException in DdJaniModelBuilder.cpp which you commented on as well